### PR TITLE
[NG] Enable cycling of alerts

### PR DESCRIPTION
**Pull request by adityarb88**
Fixes: #1695 

Tested on Chrome

nimbus-record-video-2017-11-15-13-33-36

Signed-off-by: Aditya Bhandari <adityab@vmware.com>

### CODE REVIEW

**Comment by hippee-lee** [comment]
lgtm

File: src/clarity-angular/emphasis/alert/providers/multi-alert-service.ts [hunk 1]
```diff
@@ -53,11 +53,11 @@ export class MultiAlertService {
     }
 
     next() {
-        this.current = (this.current === this.activeAlerts.length - 1) ? this.current : this.current + 1;
+        this.current = (this.current === this.activeAlerts.length - 1) ? 0 : this.current + 1;
```
Comment (Shijir) [inline src/clarity-angular/emphasis/alert/providers/multi-alert-service.ts]: 🤔  Is cycling alerts a good behavior though? What If it has only one alert? When there is only alert, users can still keep clicking on both previous and next buttons?

Comment (reddolan) [reply src/clarity-angular/emphasis/alert/providers/multi-alert-service.ts]: If there is only one alert, the cycle control should be showing.

Comment (reddolan) [reply src/clarity-angular/emphasis/alert/providers/multi-alert-service.ts]: The cycle control should also disappear after the second to last alert is dismissed, then leaving only one alert

Comment (adityarb88) [reply src/clarity-angular/emphasis/alert/providers/multi-alert-service.ts]: As @reddolan correctly pointed out, if there is only one alert, the pager disappears.

Comment (Shijir) [reply src/clarity-angular/emphasis/alert/providers/multi-alert-service.ts]: Oh so the pager disappears. Thanks for the clarification.

**Pull request merged**